ci-templates/stages.yml

```diff
@@ -149,7 +149,7 @@ stages:
               jdk: '1.12'
 
       - job: Run_TCKs
-        condition: and(eq(variables.LINUX_USE_VMS, ${{parameters.expectUseVMs}}),succeeded()) 
+        condition: and(eq(variables.LINUX_USE_VMS, ${{parameters.expectUseVMs}}),succeeded())
         timeoutInMinutes: 60
```

Comment:
please update the timeout in minutes by at least 5 minutes due to the new tests.

Reply:
Most of these changes were unrelated to the PR, my IDE auto-formatted the file after I had done my modification. So what I did here in my recent push is to increase the timeout by 5 minutes for the specific part. Could you please re-review if it is okay for you? Thanks